server/src/main/java/server/BazelServices.java

```diff
@@ -17,40 +19,42 @@
     private LanguageClient languageClient;
     private Path workspaceRoot;
 
+    private static final Logger logger = LogManager.getLogger(BazelServices.class);
```

Comment:
I think we should stick with one logger, either the `BazelServerLogger` or the `LogManager`, maybe not both. Or am I missing something?

Reply:
So, at this point the BazelServerLogger isn't used. I opted for log4j

Reply:
Is that the one we're sticking with? Should we remove the bazelserverlogger?

Reply:
Did that in the latest commit